Update GitHub links to PL repo (#13849)
Lightning-AI/lightning: src/pytorch_lightning/README.md

conda install pytorch-lightning -c conda-forge
```

> #### Install stable 1.7.x

Reviewer: ```suggestion
#### Install stable
```

> The actual status of 1.7 \[stable\] is the following:

Reviewer: ```suggestion
The actual status of stable release is the following:
```
how about we just say stable, else we might forget to update here

Author: Let's not address this here please. I don't want to go down the rabbit hole of changing the entire Readme. The intention of this PR was just to update all links with pytorch-lightning to Lightning-AI.

Author: The suggestion is great but it's not my area of expertise to update this.

[![CI base testing](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-base.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-base.yml)
[![CI complete testing](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-full-1.8.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-full-1.8.yml)
[![PyTorch & Conda](https://github.com/Lightning-AI/lightning/actions/workflows/ci-pytorch_test-conda.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci-pytorch_test-conda.yml)

> [![TPU tests](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-tpu.yml/badge.svg?branch=%27release/pytorch%27&event=push)](https://github.com/Lightning-AI/lightning/actions/workflows/ci_test-tpu.yml)

Reviewer: not 100% sure but I think this needs to be updated
https://github.com/Lightning-AI/lightning/blob/0cbfe08a41a5f517a27fd432d4770feb7bef37af/.actions/setup_tools.py#L73

here the badge refers to the version.

Reviewer: because right now it doesn't show anything
https://github.com/Lightning-AI/lightning/tree/docs/link-updates/src/pytorch_lightning#install-stable-17x

Author: And what should I do? I have no clue what all this does. I would appreciate direct help @rohitgr7 @Borda or drop this for another PR.

Reviewer: let's do a follow-up.
I already mention, we can ignore some if it get's out of scope :)

Reviewer 2: I'll make sure to update all of the badges in #13917!

the actual status of 1.6 \[stable\] is following:

![CI basic testing](https://github.com/Lightning-AI/lightning/workflows/CI%20basic%20testing/badge.svg?branch=release%2F1.5.x&event=push)
![CI complete testing](https://github.com/Lightning-AI/lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.5.x&event=push)
![PyTorch & Conda](https://github.com/Lightning-AI/lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.5.x&event=push)
![TPU tests](https://github.com/Lightning-AI/lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.5.x&event=push)

> ![Docs check](https://github.com/Lightning-AI/lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.5.x&event=push)

Reviewer: same here.

Reviewer 3: yes, all reference to `release/1.5.x` and shall be master or `release/pytorch` but then the referred function need to be updated

Author: Updated, please check if it's correct :)

Reviewer 2: I'll make sure to update all of the badges in #13917!

Install future release from the source

```bash

> pip install git+https://github.com/Lightning-AI/lightning.git@release/1.5.x --upgrade

Reviewer: ```suggestion
pip install https://github.com/Lightning-AI/lightning/archive/refs/heads/release/pytorch.zip -U
```

```

#### Install bleeding-edge - future 1.7

Install nightly from the source (no guarantees)

```bash

> pip install https://github.com/Lightning-AI/lightning/archive/master.zip

Reviewer: ```suggestion
pip install https://github.com/Lightning-AI/lightning/archive/refs/heads/master.zip -U
```
